torch/jit/_freeze.py

```diff
@@ -102,23 +100,29 @@ def forward(self, input):
 
     out = RecursiveScriptModule(torch._C._freeze_module(mod._c, preserved_attrs))
     RecursiveScriptModule._finalize_scriptmodule(out)
-    if optimize:
-        optimize_frozen_module(out)
+    optimize_frozen_module(out, optimize_numerics)
 
     return out
 
 
-def optimize_frozen_module(mod):
+def optimize_frozen_module(mod, optimize_numerics: bool = True):
```

Comment:
if you get non-portable optimizations in the future - would it be a separate `bool` flag?

Reply:
yea, that was what i was envisioning. 

`def optimize_frozen_module(mod, optimize_numerics: bool = True, non_portable = False):`